nixos/modules/services/misc/rmfakecloud.nix

```diff
@@ -0,0 +1,119 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.rmfakecloud;
+  serviceDataDir = "/var/lib/rmfakecloud";
```

Comment:
Other modules seem to be dropping options to set custom data directories because they are hard to manage when `DynamicUser` is used for the systemd service.

The effective value can still be overridden by mounting another volume or directory to `/var/lib/private/rmfakecloud`, by altering the systemd service's option, and/or by setting `extraSettings.DATADIR` to some other path. The user is then responsible to deal with potential permission issues themselves.

Reply:
> Other modules seem to be dropping options to set custom data directories because they are hard to manage when DynamicUser is used for the systemd service.

Yes, you are right, DynamicUser is problematic with existing directories...

I still find this annoying from the point that if `DATADIR` were to be set, the service would still create a useless `/var/lib/rmfakecloud` that will only be confusing. It doesn't bother me enough to block the MR because of that...